apps/sap-commerce-cloud-with-air/frontend/src/AppConfig/AppConfig.tsx

```diff
@@ -192,35 +192,37 @@ export default class AppConfig extends React.Component<Props, State> {
       <>
         {hasConfigurationOptions && (
           <Typography>
-            <Heading>Configuration</Heading>
+            <Heading style={{ marginBottom: '15px' }}>Configuration</Heading>
             <Form>
               {parameterDefinitions.map((def) => {
                 const key = `config-input-${def.id}`;
 
                 return (
-                  <TextField
-                    required={def.required}
-                    key={key}
-                    id={key}
-                    name={key}
-                    labelText={def.name}
-                    textInputProps={{
-                      width: def.type === 'Symbol' ? 'large' : 'medium',
-                      type: def.type === 'Symbol' ? 'text' : 'number',
-                      maxLength: 255,
-                    }}
-                    helpText={def.description}
-                    value={parameters[def.id]}
-                    onChange={this.onParameterChange.bind(this, def.id)}
-                  />
+                  <div style={{ marginBottom: '15px' }}>
+                    <TextField
```

Comment:
Some of the frontend issues are probably due to the fact that these are deprecated components from Forma36, we can fix in a follow up PR.

Reply:
Yup that's what i noticed as well